contrib/experimental/great_expectations_experimental/rule_based_profiler/data_assistant/statistics_data_assistant.py

```diff
@@ -0,0 +1,698 @@
+from typing import Any, Dict, List, Optional
+
+from contrib.experimental.great_expectations_experimental.rule_based_profiler.data_assistant_result import (
+    StatisticsDataAssistantResult,
+)
+from great_expectations.rule_based_profiler.config import ParameterBuilderConfig
+from great_expectations.rule_based_profiler.data_assistant import DataAssistant
+from great_expectations.rule_based_profiler.data_assistant_result import (
+    DataAssistantResult,
+)
+from great_expectations.rule_based_profiler.domain import SemanticDomainTypes
+from great_expectations.rule_based_profiler.domain_builder import (
+    ColumnDomainBuilder,
+    TableDomainBuilder,
+)
+from great_expectations.rule_based_profiler.helpers.cardinality_checker import (
+    CardinalityLimitMode,
+)
+from great_expectations.rule_based_profiler.parameter_builder import (
+    MeanTableColumnsSetMatchMultiBatchParameterBuilder,
+    MeanUnexpectedMapMetricMultiBatchParameterBuilder,
+    ParameterBuilder,
+)
+from great_expectations.rule_based_profiler.parameter_container import (
+    DOMAIN_KWARGS_PARAMETER_FULLY_QUALIFIED_NAME,
+    VARIABLES_KEY,
+)
+from great_expectations.rule_based_profiler.rule import Rule
+from great_expectations.validator.validator import Validator
+
+
+class StatisticsDataAssistant(DataAssistant):
+    """
+    StatisticsDataAssistant provides metrics for dataset exploration purposes.
```

Comment:
Could we add a Docstring here explaining how these are different than other DataAssistants (Metrics-only rather than Expectations + Metrics), and how we intend the results to be used (mentioning the vector).

Reply:
Could we add a note explaining how it can be plotted as well

Reply:
@Shinnnyshinshin This has been done.  (And plotting is standard -- no additional explanation should be needed). 
 Thanks!